src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -559,7 +559,7 @@ public HyperwalletBankCard createBankCard(HyperwalletBankCard bankCard) {
             throw new HyperwalletException("Bank Card token may not be present");
         }
         if (bankCard.getType() == null) {
-            bankCard.setType(HyperwalletTransferMethod.Type.BANK_CARD);
+            bankCard.setType(HyperwalletBankCard.Type.BANK_CARD);
```

Comment:
Since we are changing the type here to HyperwalletBankCard, do we need to remove these enum values(BANK_CARD, PAYPAL_ACCOUNT) from HyperwalletTransferMethod.java?

Reply:
This is a good point. But HyperwalletTransferMethod is being used in other places. We need to do thorough testing if we are removing this. Its better to invest that time in V4 than in V3. So I didnt touch HyperwalletTransferMethod.